Author: Andreas Tille <tille@debian.org>
Last-Update: Tue, 29 Mar 2016 22:11:29 +0200
Description: Fix clean target of Makefile

--- a/Makefile
+++ b/Makefile
@@ -84,10 +84,10 @@ test_gunzip : test_gunzip.o libgzstream.
 	${CXX} -o test_gunzip test_gunzip.o ${LDFLAGS}
 
 clean :
-	rm *.o
+	rm -f *.o
 
 cleanall :
-	rm *.o libgzstream.a test_gzip test_gunzip
+	rm -f *.o libgzstream.a test_gzip test_gunzip
 
 uninstall:
 	rm /usr/include/gzstream.h /usr/lib/libgzstream.a
